workflows/theiacov/wf_theiacov_illumina_pe.wdl

```diff
@@ -157,96 +171,133 @@ workflow theiacov_illumina_pe {
           call abricate.abricate_flu {
             input:
               assembly = select_first([irma.irma_assembly_fasta]),
-              samplename = samplename,
-              nextclade_flu_h1n1_ha_tag = nextclade_flu_h1n1_ha_tag,
-              nextclade_flu_h1n1_na_tag = nextclade_flu_h1n1_na_tag,
-              nextclade_flu_h3n2_ha_tag = nextclade_flu_h3n2_ha_tag,
-              nextclade_flu_h3n2_na_tag = nextclade_flu_h3n2_na_tag,
-              nextclade_flu_vic_ha_tag = nextclade_flu_vic_ha_tag,
-              nextclade_flu_vic_na_tag = nextclade_flu_vic_na_tag,
-              nextclade_flu_yam_tag = nextclade_flu_yam_tag,
+              samplename = samplename
+          }
+          call set_organism_defaults.organism_parameters as set_flu_na_nextclade_values {
+            input:
+              organism = organism_parameters.standardized_organism,
+              flu_segment = "NA",
+              flu_subtype = irma.irma_subtype,
+              # including these to block from terra
+              reference_gff_file = reference_gff,
+              reference_genome = reference_genome,
+              genome_length = genome_length,
+              nextclade_ds_reference = nextclade_dataset_reference,
+              nextclade_ds_tag = nextclade_dataset_tag,
+              nextclade_ds_name = nextclade_dataset_name,     
+              vadr_max_length = vadr_max_length,
+              vadr_options = vadr_options,
+              primer_bed_file = primer_bed,
+              pangolin_docker_image = pangolin_docker_image,
```

Comment:
can we remove `pangolin_docker_image` from this input block (and additionally the input block below for `set_flu_ha_nextclade_values`?)

Looks like an optional input for organism param subwf so might be good to remove

Reply:
Ah, now I see the comment. disregard!